schemas/test/en/test_placeholder_first_non_empty_item_cross_section_dependencies.json

```diff
@@ -0,0 +1,291 @@
+                            ]
+                        },
+                        {
+                            "id": "date-entry-block",
+                            "type": "Question",
+                            "question": {
+                                "id": "date-entry-question",
+                                "title": "What are the dates of the period that you will be reporting for?",
+                                "guidance": {
+                                    "contents": [
+                                        {
+                                            "description": "Enter a date between 24 June 2023 and 2 August 2023"
+                                        }
+                                    ]
+                                },
+                                "type": "DateRange",
+                                "answers": [
+                                    {
+                                        "id": "date-entry-answer-from",
+                                        "type": "Date",
+                                        "mandatory": true,
+                                        "label": "From",
+                                        "minimum": {
+                                            "value": {
+                                                "source": "metadata",
+                                                "identifier": "ref_p_start_date"
+                                            },
+                                            "offset_by": {
+                                                "days": -19
+                                            }
+                                        }
+                                    },
+                                    {
+                                        "id": "date-entry-answer-to",
+                                        "type": "Date",
+                                        "mandatory": true,
+                                        "label": "To",
+                                        "maximum": {
+                                            "value": {
+                                                "source": "metadata",
+                                                "identifier": "ref_p_end_date"
+                                            },
+                                            "offset_by": {
+                                                "days": 20
+                                            }
+                                        }
+                                    }
+                                ],
+                                "period_limits": {
+                                    "minimum": {
+                                        "days": 10
+                                    },
+                                    "maximum": {
+                                        "days": 50
+                                    }
+                                }
```

Comment:
We should remove these from the test schema as it make it harder to test.

Reply:
Removed 👍